backend/node/genvm/base.py

```diff
@@ -258,24 +279,51 @@ async def run_contract(
                 else:
                     function_to_run(*args)
             except Exception as e:
-                print("\nError running contract", e)
-                print(f"\n{traceback.format_exc()}")
+                trace = traceback.format_exc()
                 error = e
```

Comment:
```suggestion
            except Exception as error:
                trace = traceback.format_exc()
```

Reply:
same- actually need the remapping of `error`  here because it is used down below when returning the receipt